groovy/postInit/chemistry/organic_chemistry/petrochemistry/Lubricants.groovy

```diff
@@ -39,8 +37,8 @@ PHASE_SEPARATOR = recipemap('phase_separator')
     // Straight run lubricating oil
 
     MIXER.recipeBuilder()
-        .fluidInputs(fluid('dichloroethane') * 1000)
-        .fluidInputs(fluid('dichloromethane') * 1000)
+        .fluidInputs(fluid('butanone') * 1000)
```

Comment:
https://link.springer.com/article/10.1007/BF02694095 implies toluene could be used just as well.

Reply:
there are a variety of solvents but i cant bother to implement so many.